### PR TITLE
calculate the correct x pixel in projection accu

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[Projektion sollte x-pixel index richtig berechnen.](https://trello.com/c/c8VltUSh/791-projektion-sollte-x-pixel-index-richtig-berechnen)

**Comment by jdroenner** [comment]
[Projection Parallelisierung checken](https://trello.com/c/YsSWj2Bv/789-projection-parallelisierung-checken)

File: operators/src/adapters/raster_subquery/raster_subquery_reprojection.rs [hunk 1]
```diff
@@ -306,34 +306,38 @@ where
...
-                            let lookup_value = tile.pixel_value_at_coord_unchecked(coord);
-                            *pixel = lookup_value;
+            .for_each(|(chunk_idx, chunk_slice)| {
+                let chunk_start_y = chunk_idx * par_chunk_split;
+                chunk_slice
+                    .iter_mut()
+                    .enumerate()
+                    .for_each(|(lin_idx, pixel)| {
+                        let x_idx = lin_idx % axis_size_x;
+                        let y_idx = lin_idx / axis_size_x + chunk_start_y;
```
Comment (michaelmattig) [inline operators/src/adapters/raster_subquery/raster_subquery_reprojection.rs:330]: you calculate the `y_idx` for each pixel. Isn't the way I do it in the interploation operator more efficient? (Calculating start and end y and then iterating over slices with precalculated y index)

Also: can you please add a test that would have failed without this change. It worries me that this change doesn't break any tests...

Comment (jdroenner) [reply operators/src/adapters/raster_subquery/raster_subquery_reprojection.rs]: i added debug_asserts so the error in question should show up in testing

**Comment by michaelmattig** [comment]
bors merge

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**